### PR TITLE
Remove accounting config

### PR DESCRIPTION
**Pull request by mberhault**
Work towards #2090

Remove accounting config/keys/endpoints/cli/etc...

There is currently nothing in the accounting config, and no exact plans.
When this emerges, we'll move it into the system database alongside
other configs.

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM

File: config/prefix.go [hunk 1]
```diff
@@ -83,8 +83,8 @@ func (p PrefixConfigMap) Less(i, j int) bool {
 //
 // These additional entries allow for simple lookups by prefix and
 // provide a way to split a range by prefixes which affect it. This
-// last is necessary for accounting and zone configs; ranges must not
-// span accounting or zone config boundaries.
+// last is necessary for zone configs; ranges must not
```
Comment (tamird) [inline config/prefix.go]: "This last"?

File: storage/engine/mvcc_test.go [hunk 2]
```diff
@@ -1745,10 +1745,11 @@ func TestValidSplitKeys(t *testing.T) {
 		{proto.Key("\x00\x00meta2\xff\xff"), false},
 		{proto.Key("\x00\x00meta3"), true},
 		{proto.Key("\x00\x01"), true},
+		// Deprecated accounting config key. Valid split.
```
Comment (tamird) [inline storage/engine/mvcc_test.go]: shouldn't this be two lines down?

**Comment by mrtracy** [comment]
LGTM

**Comment by tamird** [comment]
LGTM

File: config/prefix.go (hunk 1 above)
Comment (MBerhault) [inline config/prefix.go]: uh. yeah. same as it was before, refers to "split a range by prefixes which affect it"

File: storage/engine/mvcc_test.go (hunk 2 above)
Comment (MBerhault) [inline storage/engine/mvcc_test.go]: it covers the entire accounting-related ranges.

File: config/prefix.go (hunk 1 above)
Comment (tamird) [inline config/prefix.go]: i don't get it; this last what?

Comment (MBerhault) [inline config/prefix.go]: aka: "the latter"

**Pull request merged**